aplt/scenarios.py

```diff
@@ -249,6 +290,80 @@ def _test_multiple_spawn():
     yield spawn("aplt.scenarios:basic, 1, 1, 0")
 
 
+def _api_test():
```

Comment:
Definitely, the test scenarios by the way are intended for use by ap-loadtester tests to verify the program functions correctly, not to test the service. I should have named it better to reflect that since 'test' is getting awfully overloaded now, ie, "Internal APLT Tests".

The complete spawn scenario func shouldn't start with a _ as well.